build.gradle

```diff
@@ -221,14 +221,15 @@ tasks.withType(Test) {
 tasks.withType(Javadoc) {
     title = "${project.name} ${project.version}"
     configure(options) {
+        source = JavaVersion.VERSION_1_8
```

Comment:
What is this meant to do?  `source` on Javadoc tasks should be files, but here we're passing in a version string.  Why?

Reply:
OK, found it - Gradle docs are hard to navigate sometimes: https://docs.gradle.org/current/javadoc/org/gradle/external/javadoc/CoreJavadocOptions.html#getSource--

tl;dr this sets the language version allowable by the javadoc process.